app/routes/index.js

```diff
@@ -1,8 +1,7 @@
 import Route from '@ember/routing/route';
-import { get } from '@ember/object';
 
 export default Route.extend({
   redirect() {
-    this.transitionTo('version', get(this.modelFor('application'), 'currentVersion'));
```

Comment:
There doesn't seem to be a corresponding acceptance test for this. Can we add it?